### PR TITLE
Server controller

### PR DESCRIPTION
**Pull request by y-abs**
## What does this PR do ?

Add server controller documentation

### CODE REVIEW

File: doc/2/controllers/server/get-config/snippets/get-config.test.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name: server#getConfig
+description: Returns the current Kuzzle configuration.
+hooks:
+  before:
+  after:
+template: print-result
+expected: ^{.*}$
```
Comment (jenow) [inline doc/2/controllers/server/get-config/snippets/get-config.test.yml]: Uh I am not convinced by the strength of this test :sweat_smile:

Comment (y-abs) [reply doc/2/controllers/server/get-config/snippets/get-config.test.yml]: The result is a very large object, I can change it to match some fields in deed

Comment (jenow) [reply doc/2/controllers/server/get-config/snippets/get-config.test.yml]: Yes so at least it's would be different than an exception occurring :)

**Review by jenow — APPROVED** [review]

File: doc/2/controllers/server/now/snippets/now.test.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+name: server#now
+description: Fetch the current server timestamp, in Epoch-millis format.
+hooks:
+  before:
+  after:
+template: print-result
+expected: ^[0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1]) (2[0-3]|[01][0-9]):[0-5][0-9]:[0-5][0-9].[0-9][0-9][0-9]$
```
Comment (alexandrebouthinon) [inline doc/2/controllers/server/now/snippets/now.test.yml:7]: IMO, we should return a proper UNIX timestamp which is more flexible and permit the users to format it :slightly_smiling_face:

**Review by alexandrebouthinon — CHANGES_REQUESTED** [review]

File: doc/2/controllers/server/now/snippets/now.test.yml (hunk 2 above)
Comment (Aschen) [reply doc/2/controllers/server/now/snippets/now.test.yml:7]: The method return a high level DateTime object which is only printed like this

**Review by alexandrebouthinon — APPROVED** [review]

**Pull request merged**